js_test.sh

```diff
@@ -3,17 +3,17 @@ export TMP_FILE=$(mktemp)
 
 if [[ ! -z "$COVERAGE" ]]
 then
-    export CMD="node ./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha --"
+    export CMD="node ./node_modules/nyc/bin/nyc.js --reporter=html mocha "
 elif [[ ! -z "$CODECOV" ]]
 then
-    export CMD="node ./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha -- --report lcovonly -R spec"
```

Comment:
Now that we're not using the CLI that ships with the `istanbul` package, I think we can remove `istanbul` from the dependency list in `package.json`. [`nyc` doesn't depend on it.](https://github.com/istanbuljs/nyc/blob/master/package.json#L74-L102)